session/bootstrap.go

```diff
@@ -985,11 +994,14 @@ const (
 	// version 175
 	//   add column `scan_predicate_hash` to `mysql.stats_meta`.
 	version175 = 175
+	// vresion 180
```

Comment:
Why here is 180，not 176？

Reply:
In https://github.com/pingcap/tidb/pull/48979, I use v180. I didn't change this for the demo in this branch.